bot/extensions/fun/fun_cog.py

```diff
@@ -39,6 +42,36 @@ async def quote_command(self, ctx):
 
         await ctx.send(embed=embed)
 
+    async def display_color(self, color_img, ctx):
+        color_img.save('color.png')
+
+        embed = Embed(
+            color=self.bot.default_color,
+            description='Here goes your color!'
+        )
+        file = File('color.png')
+        await ctx.send(embed=embed, file=file)
+
+        os.remove('color.png')
+    
+
+    @command(name='rgb', help='''Displays the RGB color entered by the user.''')
```

Comment:
You don't need the three `'''`. You can just use single quote (') or double quotes (")

Reply:
Fixed!